tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 minversion = 1.4.2
-envlist = linters
+envlist = linters,tools
```

Comment:
we also need a new zuul jobs, for this. Could you add it?

Reply:
we can then add filters match to run only when tox.ini, tools/missing_keys and nodepool directory changes

Reply:
ok. I will add that.